app/index.js

```diff
@@ -39,7 +39,7 @@ module.exports = generators.NamedBase.extend({
     }, {
       type: 'input',
       name: 'userName',
-      message: 'What is your name? (Used for the package.json.)',
+      message: 'What is your name? (Used for the package.json. Capitalize the first character)',
```

Comment:
@derekpeterson But I think this should be added at the name of project part, what do you think